.expeditor/scripts/end_to_end/run_e2e_test_core.ps1

```diff
@@ -204,6 +204,15 @@ function Get-Leader($Remote, $ServiceGroup) {
     }
 }
 
+Function Invoke-BuildAndInstall($PackageName) {
+    hab pkg build test/fixtures/$PackageName
```

Comment:
This is just a question of curiosity. Do we prefer building packages in the tests or downloading them from bldr? I want to try and keep what I do when writing e2e tests consistent if possible.

Reply:
personally I do. I find that it just makes iterating over writing the e2e test easier since I can make changes to the package plans and then just run the test without having to upload anywhere.